tests/integration_tests/test_examples.py

```diff
@@ -1,39 +1,48 @@
+"""This is an integration test to make sure scripts from examples/
+work when executing `./examples/**/*.py`.
 """
-This is an integration test to make sure scripts from examples/
-work when running `python examples/xx/xxx.py`.
-"""
-from garage.experiment import LocalRunner, run_experiment
-from garage.np.baselines import LinearFeatureBaseline
-from garage.tf.algos import VPG
-from garage.tf.envs import TfEnv
-from garage.tf.policies import CategoricalMLPPolicy
+import os
+from pathlib import Path
+import subprocess
+
+import pytest
+
+EXAMPLES_ROOT_DIR = Path('examples/')
+
+
+def enumerate_algo_examples():
+    """Return a list of paths for all algo examples
+
+    Returns:
+        List[str]: list of path strings
+
+    """
+    return map(str, EXAMPLES_ROOT_DIR.glob('*/**/*.py'))
```

Comment:
hmm why not `glob('**.py')`? + a blacklist -- then you don't need to refactor if the examples folder gets reorganized.

Reply:
That's what I did earlier. Maybe I misunderstood this comment: https://github.com/rlworkgroup/garage/pull/987#discussion_r342887200

Reply:
i like the updated version, i'm just suggesting:
```python
NON_ALGORITHM_EXAMPLES = [
    EXAMPLES_ROOT_DIR / 'foo.py',
    EXAMPLES_ROOT_DIR / 'bar.py',
]

def enumerate_algo_examples():
    all_examples = EXAMPLES_ROOT_DIR.glob('**/*.py')
    return [e for e in all_examples if e not in NON_ALGORITHM_EXAMPLES]
```

AFAICT your current test would not catch `examples/algo_example.py` because it assumes that algo examples are always inside a subdir

Reply:
Calling it a blacklist doesn't seem appropriate if some of the files on that list have their own tests. So I am thinking a function local "exclude" list instead

Reply:
Yes, this seems better. I'll change to this.